packages/edit-site/src/components/sidebar-dataviews/custom-dataviews-list.js

```diff
@@ -43,6 +56,43 @@ function CustomDataViewItem( { dataviewId, isActive } ) {
 			isActive={ isActive }
 			isCustom="true"
 			customViewId={ dataviewId }
+			suffix={
+				<DropdownMenu
```

Comment:
This results in nested `button` tags, which is invalid HTML. Additionally when we click the DropDown it also triggers the `click` event in the list item.